src/main/js/component/AboutInfo.js

```diff
@@ -57,59 +57,32 @@ class AboutInfo extends React.Component {
         );
     }
 
-    createProviderTableData(globalDescriptors) {
+    createTableData(userBasedDescriptors, descriptors) {
         const data = [];
-        for (let key in globalDescriptors) {
-            const descriptor = globalDescriptors[key];
-            if (!data.find(existing => existing.urlName === descriptor.urlName)) {
+        for (let key in descriptors) {
+            const descriptor = descriptors[key];
```

Comment:
For each descriptor (providers or channels) check the users list of descriptors and see if they have permission for a Global UI context.  If they do then navigate will be true and the item in the table will be a link not just the name of the descriptor.

Reply:
I like this. 

A comment for the sake of other reviewers who may not be comfortable with this: 
> I think metadata should be available to any authenticated user, but the UI should be responsible for determining what metadata it displays. Only the `field_values` table contains sensitive content that we care about securing. As an open source project, all of this metadata is available here on GitHub anyway, so why complicate our API by trying to restrict it on a per-user basis.